core/templates/core/clients.html

```diff
@@ -18,6 +18,7 @@
 {% block extra_js %}
     <script type="riot/tag" src="{% static 'core/tags/clients.tag' %}"></script>
     <script type="riot/tag" src="{% static 'core/tags/client.tag' %}"></script>
+    <script type="riot/tag" src="{% static 'core/tags/pager.tag' %}"></script>
```

Comment:
We could make another core template that is called "paginated" that we extend but that seems like it would be just as much work if not more. I think this is fine.